extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/runtime/WebSocketConnectionImpl.java

```diff
@@ -106,6 +110,14 @@ public Uni<Void> sendPing(Buffer data) {
         return UniHelper.toUni(webSocket.writePing(data));
     }
 
+    void sendAutoPing() {
+        webSocket.writePing(Buffer.buffer("ping")).onComplete(r -> {
+            if (r.failed()) {
+                LOG.errorf(r.cause(), "Unable to send auto-ping: %s", this);
```

Comment:
Is it really an error? I would just use "warning" (for example, the connection got cut, not a big deal).

Reply:
Well, the timer is removed when the connection is closed but you're right that a warning might be enough.